dockers/docker-database/docker-database-init.sh

```diff
@@ -1,13 +1,26 @@
 #!/usr/bin/env bash
 
-mkdir -p /var/run/redis/sonic-db
-if [ -f /etc/sonic/database_config.json ]; then
-    cp /etc/sonic/database_config.json /var/run/redis/sonic-db
+REDIS_DIR=/var/run/redis$NAMESPACE_ID
+mkdir -p $REDIS_DIR/sonic-db
+
+if [ -f /etc/sonic/database_config$NAMESPACE_ID.json ]; then
+    cp /etc/sonic/database_config$NAMESPACE_ID.json $REDIS_DIR/sonic-db/database_config.json
 else
-    cp /etc/default/sonic-db/database_config.json /var/run/redis/sonic-db
+    python -c "import sys;import jinja2;loader = jinja2.FileSystemLoader('/usr/share/sonic/templates/');env = jinja2.Environment(loader=loader, trim_blocks=True);template = env.get_template('database_config.json.j2');print(template.render(NAMESPACE=sys.argv[1]))" "$NAMESPACE_ID" > $REDIS_DIR/sonic-db/database_config.json
```

Comment:
line too long.

Reply:
I am looking at other options to do this same ..using **sonic-cfggen**. But doesn't work in this scenario --  as the "-t" option uses SonicV2Connector class which in turn uses the database_config.json which we are trying to generate here.

sonic-cfggen -t /usr/share/sonic/templates/database_config.json.j2 -a "{\"NAMESPACE\":\"$NAMESPACE_ID\"}" > $REDIS_DIR/sonic-db/database_config.json

sonic-cfggen -t /usr/share/sonic/templates/database_global.json.j2 -a "{\"NAMESPACE_CNT\":$NAMESPACE_COUNT,\"PREFIX\":\"$NAMESPACE_PREFIX\"}" > $REDIS_DIR/sonic-db/database_global.json.

So I have kept it the same and made the long statement aligned, will that suffice ?

Reply:
I would try to use j2cli if I could make this line short.

Reply:
I don't see j2cli available in the device .. in host and in docker -- looks to me that it is not installed in the SONIC file system

Reply:
Have installed jinja and j2cli in the docker-base-stretch so that it is available for all dockers.  Updated the code to use j2.